inspectit-oce-core/src/main/resources/config/default.yml

```diff
@@ -17,6 +17,20 @@ inspectit:
       # if true the directory will be watched for changes. When changes occur, the configuration is automatically reloaded
       watch: true
 
+  #settings for configuring OpenCensus stats and trace exporters
```

Comment:
do you think we need some kind of ultimate prop for metrics and tracing, for example:

`inspectit.metrics.enabled=true`

When this would be false nothing related to metrics would be done, like an electricity switch.. Then you don't need to configure set of other metric related things like no exporter, no this metric, no that metrics, etc..